lib/cocoapods/user_interface.rb

```diff
@@ -62,6 +62,7 @@ def section(title, verbose_prefix = '', relative_indentation = 0)
         self.indentation_level += relative_indentation
         self.title_level += 1
         yield if block_given?
+      ensure
```

Comment:
+1 on this change, not clear why it's in this pr tho...

Reply:
+1 separate PR coming up.

Reply:
PR https://github.com/CocoaPods/CocoaPods/pull/7634